examples/eap/legacy-s2i-source-build/helm.yaml

```diff
@@ -0,0 +1,19 @@
+build:
+  uri: https://github.com/jboss-container-images/jboss-eap-8-openshift-image
+  mode: s2i
+  contextDir: examples/eap/legacy-s2i-source-build
+  ref: eap8-beta-dev
+  s2i:
+    version: latest
```

Comment:
this will be *slightly* different for the EAP Helm chart under development at https://github.com/jbossas/eap-charts/pull/29 with something like:

```yaml
build:
  uri: ...
  s2i:
    jdk: "11"
```

Reply:
let's keep the working wildfly chart until then...

Reply:
For now these Helm charts are more place holder, they can't be used, we still don't have the artifacts deployed in MRRC.